helm/values.yaml

```diff
@@ -11,13 +11,13 @@ initialDelaySeconds: 60
 resetOnConfigChange: true
 
 cloudProvider:
-  dockerRegistryUrl: 'my-registry'
```

Comment:
disable lint/prettier and such for helm!!!
on other services it caused lots of unneeded changes that cause conflicts and sometimes even broke the chart

Reply:
Fixed and added prettier ignore.